doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

```diff
@@ -0,0 +1,500 @@
+      set output frequency  = 100
+      set time end          = 20
+      set time step         = 0.001
+      set output path       = ./output/
+    end
+
+Since the alumina particles are more than 3 times denser than alginate particles, the pseudo-steady state is reached after very different times (according to Ferreira *et al*. `[1] <https://doi.org/10.1016/j.powtec.2023.118652>`_ 4 and 10 seconds of real time, respectively). Because of this, we use ``set time end = 35`` for the alginate.
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. The values are meant to reproduce the characteristics of water at :math:`30 \: ^oC`.
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.0000008379
+        set density             = 997
+      end
+    end
+
+
+Initial Conditions
+~~~~~~~~~~~~~~~~~~
+
+For the initial conditions, we choose zero initial conditions for the velocity. 
+
+.. code-block:: text
+
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+          set Function expression = 0; 0; 0; 0
+      end
+    end
+ 
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on the walls (ID = 0) and an inlet velocity of 0.157033 m/s at the lower face of the bed (ID = 1).
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 2
+      subsection bc 0
+        set id   = 0
+        set type = slip
+      end
+      subsection bc 1
+        set id   = 1
+        set type = function
+        subsection u
+          set Function expression = 0.157033
+        end
+        subsection v
+          set Function expression = 0
+        end
+        subsection w
+          set Function expression = 0
+        end
+      end
+    end
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ .
+
+Void Fraction
+~~~~~~~~~~~~~~~
+
+We choose the `particle centroid method (PCM) <../../../parameters/unresolved-cfd-dem/void-fraction.html>`_ to calculate void fraction. The ``l2 smoothing factor`` we choose is around the square of twice the particle’s diameter, as in the other examples.
+ 
+.. code-block:: text
+
+    subsection void fraction
+      set mode                = pcm
+      set read dem            = true
+      set dem file name       = dem
+      set l2 smoothing factor = 0.000028387584
+      set bound void fraction = false
+    end
+
+.. note::
+    Note that void fraction is not bound in this case. The particles used in this example obligates us to use a very coarse mesh. Bounding void fraction lead to instability in the present case.
```

Comment:
```suggestion
    Note that void fraction is not bound in this case. The size particles used in this example obligates us to use a very coarse mesh. Bounding void fraction would lead to instability in the present case.
```